### PR TITLE
Import ROP code

### PR DESCRIPTION
**Pull request by zonca**
Shared branch so I can also help improve this (fix Travis tests for example)

### CODE REVIEW

File: iris_pipeline/readout/generate_ramp.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        gain = np.ones(shape=(nrows, ncols), dtype=np.float64) * gain
+        err = np.ones(shape=(nints, ngroups, nrows, ncols), dtype=np.float64)
+        data = np.zeros(shape=(nints, ngroups, nrows, ncols), dtype=np.float64)
+        data=arr
+        pixdq = np.zeros(shape=(nrows, ncols), dtype= np.float64)
+        read_noise = np.full((nrows, ncols), readnoise, dtype=np.float64)
+        gdq = np.zeros(shape=(nints, ngroups, nrows, ncols), dtype=np.int32)
+        model1 = RampModel(data=data, err=err, pixeldq=pixdq, groupdq=gdq, times=times)
+        model1.meta.instrument.name='MIRI'
+        model1.meta.instrument.detector='MIRIMAGE'
```
Comment (zonca) [inline iris_pipeline/readout/generate_ramp.py:27]: @arunsurya77 both the `test_ramp.fits` file generated by this file and the `sample_ramp.fits` file included in the repo have metadata about MIRI.
Do you have a version of this that instead generates a IRIS ramp?
You should have access to the `import_rop` branch so you can directly commit into this pull request.

Comment (arunsurya77) [reply iris_pipeline/readout/generate_ramp.py:27]: the generate_ramp.py is actually an oldfile that I used sometime before that is obsolete. I forgot to add the test_ramp.fits  file. i will upload it in figshare and send the link

Comment (zonca) [reply iris_pipeline/readout/generate_ramp.py:27]: I guess you must have an updated `generate_ramp.py` that you used to create the new `test_ramp.fits`, can you please add it to the repo as well?

**Comment by zonca** [comment]
@arunsurya77 do you have a more meaningful test case that actually tests that the sampling works as expected?

**Comment by zonca** [comment]
ok, tests on travis are passing!

**Comment by zonca** [comment]
merging this, I'll open a separate issue about testing

**Pull request merged**